helloworld-mutual-ssl-secured/configure-client-truststore.cli

```diff
@@ -0,0 +1,11 @@
+#extract the certificate for the server keystore into a file called serverCert.crt
```

Comment:
Export the server's certificate to a file called serverCert.crt

Reply:
fixed!